packages/contracts-bedrock/test/scripts/L2Genesis.t.sol

```diff
@@ -139,4 +177,81 @@ contract L2Genesis_Run_Test is L2Genesis_TestInit {
         testFactories();
         testForks();
     }
+
+    /// @dev Modifier to set up the input for L2Genesis with CGT enabled.
+    modifier setInputCGTEnabled() {
```

Comment:
Nit - probably cleaner to add a `setUp` function with a default input and then just have the CGT tests change the input. Also same comment as in other file about modifier vs helper function